integration-test/extensionconfig/public-azurechina.json

```diff
@@ -0,0 +1,26 @@
+{
+	"docker" : {
+		"port" : "2376",
+		"options" : [
+			"--label",
+			"foo=bar"
+			]
+	},
+	"compose" : {
+		"envdump" : {
+			"image" : "mirror.azure.cn:5000/library/busybox",
```

Comment:
@ahmetalpbalkan that's good point. Now mirror.azure.cn:5000 for docker private registry is only for dev/test, not for production. Put on another host(actually two VMs as load balancer) would require more hardware budget, would do that if we have decided to put it to production. Thanks for your suggestion.